data/defaults.csv

```diff
@@ -0,0 +1,6 @@
+env_setup_commands,pip install --upgrade pip<19
+env_setup_commands,poetry install
+env_setup_commands,pre-commit install
+self_check_commands,pre-commit run -a
```

Comment:
I think you know what I'm going to say here.

Reply:
see above re: poetry